src/Tools/dotnet-dump/Program.cs

```diff
@@ -108,7 +108,7 @@ public static Task<int> Main(string[] args)
         private static Option RunCommand() =>
             new Option(
                 aliases: new[] { "-c", "--command" }, 
-                description: "Runs the command on start. Multiple instances of this parameter can be used in an invocation to chain commands. Commands will get run in the order that they are provided on the command line. If you want to exit analysis after the commands, you must explicitly add an 'exit' command at the end.") 
+                description: "Runs the command on start. Multiple instances of this parameter can be used in an invocation to chain commands. Commands will get run in the order that they are provided on the command line. If you want to dotnet dump to exit after the commands, your last command should be 'exit'.") 
```

Comment:
Suggest "If you want dotnet dump to exit..." , removing the "to" before "dotnet".

Reply:
oh, nuts! I do'nt have write access to this repo so I can't fix it.